src/jose/Converter.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+/**
+ * Module Dependencies
+ * @ignore
+ */
+const JWT = require('./JWT')
+const JWD = require('./JWD')
+
+/**
+ * Converter
+ */
+class Converter {
+
+  /**
+   * toJWD
+   *
+   * @description
+   * Convert a JWT to JWD
+   *
+   * @param  {JWT} data
+   * @return {JWD}
+   */
+  static toJWD (data) {
+    return new JWD(data)
+  }
+
+  /**
+   * toJWT
+   *
+   * @description
+   * Convert a JWD to JWT
+   *
+   * @param  {JWD} data
+   * @return {JWT}
+   */
+  static toJWT (data) {
+    return new JWT(data)
```

Comment:
Makes sense, a reasonable approach. The other alternative to handling the circular dependency, though, is to import the relevant class JWT / JWD *inside* the static method. (This is the approach I've taken in RDFLib in the case of circular deps.) I'm not necessarily against an external Converter class, just wanted to bring up the alternative.